authentication/models.py

```diff
@@ -1,3 +1,55 @@
 from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
+from django.contrib.auth.models import PermissionsMixin
+from validation.validate_password import validate_password_long, validate_password_include_symbols
 
-# Create your models here.
+
+class CustomUserManager(BaseUserManager):
+
+    def create_user(self, person_email, password=None, **extra_fields):
+        person_email = self.normalize_email(person_email)
+        extra_fields.setdefault("is_staff", False)
+        extra_fields.setdefault("is_superuser", False)
```

Comment:
Is it possible to remove setdefault? The values are already set by default in CustomUser model

Reply:
User create methods improved, setdefault removed.